wandb/sdk/wandb_artifacts.py

```diff
@@ -466,13 +467,32 @@ def verify(self, root: Optional[str] = None):
             "Cannot call verify on an artifact before it has been logged or in offline mode"
         )
 
-    def save(self) -> None:
+    def save(self, **init_kwargs) -> None:  # type: ignore
         if self._logged_artifact:
             return self._logged_artifact.save()
-
-        raise ValueError(
-            "Cannot call save on an artifact before it has been logged or in offline mode"
-        )
+        else:
+            if wandb.run is None:
+                kwargs = {k: v for k, v in init_kwargs.items()}
```

Comment:
Curious if there's a reason for making a copy of the kwargs?  I would think just using **kwargs in the method would be ok.  Definitely a nit.